Update archives.md (#18293)
DataDog/documentation: content/en/logs/log_configuration/archives.md

Set up the [Azure integration][1] within the subscription that holds your new storage account, if you haven't already. This involves [creating an app registration that Datadog can use][2] to integrate with.

> **Note:** Archiving to Azure ChinaCloud, GermanyCloud and GovCloud are not supported. We only support archiving logs to Azure public cloud.

Reviewer: ```suggestion
**Note:** Archiving to Azure ChinaCloud, GermanyCloud, and GovCloud is not supported.
```

Author: Thanks for feedback. I have committed your suggestion.
